menuflow/example-config.yaml

```diff
@@ -35,6 +35,10 @@ menuflow:
         start: 1
         end: 3
 
+    # It defines how much time have to wait before receive a message in a room,
```

Comment:
Cambiar por 
```
# It defines how much time to wait between two consecutive messages from the customer to process the message
# it's to avoid sending too much messages to the customer
```